config/database.yml

```diff
@@ -10,24 +10,24 @@ default: &default
 
 development:
   <<: *default
-  database: flextentions_dev
+  database: flextensions_dev
```

Comment:
not necessarily a problem for just the dev database but please use correct spelling "flextensions" throughout.  (== "flexible" + "extensions")